src/patternfly/components/Alert/alert.scss

```diff
@@ -132,7 +132,6 @@
 
   &.pf-m-inline {
     --pf-c-alert--BoxShadow: var(--pf-c-alert--m-inline--BoxShadow);
-    --pf-c-alert__title--Color: var(--pf-c-alert--m-inline__title--Color);
     --pf-c-alert__icon--BackgroundColor: var(--pf-c-alert--BackgroundColor);
```

Comment:
You should probably remove the variable then also, line 63.

Reply:
:)